tutorials/examples/preprocess_imgs.py

```diff
@@ -0,0 +1,92 @@
+"""
+This script illustrates how to preprocess experimental CryoEM images
+before performing reconstructing 3D map.
+"""
+
+import logging
+import matplotlib.pyplot as plt
+
+from aspire.source.relion import RelionSource
+from aspire.estimation.noise import WhiteNoiseEstimator
+
+logger = logging.getLogger('aspire')
+
+starfile_in = '/tigress/junchaox/CryoEMdata/empiar10028/shiny_2sets.star'
+data_folder = '/tigress/junchaox/CryoEMdata/empiar10028/'
+pixel_size = 1.34
+num_imgs = 1024
+
+logger.info('This script illustrates how to preprocess experimental CryoEM images')
+logger.info(f'Read in images from {starfile_in} and preprocess the images')
+source = RelionSource(
+    starfile_in,
+    data_folder,
+    pixel_size=pixel_size,
+    max_rows=num_imgs
+)
+
+logger.info('Obtain original images')
+imgs_od = source.images(start=0, num=num_imgs).asnumpy()
+
+logger.info('Perform phase flip to input images')
+starfile_out = f'{data_folder}/preprocess/shiny_2sets_phase_flip.star'
+source.phase_flip()
+imgs_pf = source.images(start=0, num=num_imgs).asnumpy()
+source.save(starfile_out, batch_size=512, save_mode='single', overwrite=False)
```

Comment:
Let's not save the output. There is no need if our purpose is to demonstrate the preprocessing steps.

Also, why do we have to specify `batch_size` and `save_mode`. Are these not provided by default in the `save` method?

Reply:
Removed the save statement in new commit 49330e9.